docs/contributor/arch/013-resource-attributes-enrichment-for-non-workload-metrics.md

```diff
@@ -31,10 +31,10 @@ An `Input Pipeline`, which contains the `otlp` receiver and `memory_limiter` pro
 An `Attributes Enrichment Pipeline`, which contains the `k8sattributes` and `transform/resolve-service-name` processors.
 An `Output Pipeline`, which contains the rest of the processors and the `otlp` exporter.
 
-The metrics will be routed to the `Attributes Enrichment Pipeline` only if they are workload metrics.
+The metrics will be routed to the `Attributes Enrichment Pipeline` only if they are workload metrics. This can be done by checking if a resource attribute (like `skip-enrichment`) is set `true`.
```

Comment:
I'm not sure what this means.

```suggestion
The metrics will be routed to the `Attributes Enrichment Pipeline` only if they are workload metrics. To set up the routing, check if a resource attribute (like `skip-enrichment`) is set `true`.
```

or 

```suggestion
The metrics will be routed to the `Attributes Enrichment Pipeline` only if they are workload metrics. To check whether they are workload metrics, check if a resource attribute (like `skip-enrichment`) is set `true`.
```

Basically, what is the "this" that "can be done" (by whom)?

Reply:
"this" refers to the conditional routing
It is done by the routing connector

Reply:
I changed the structure of the sentence. Can you have another look?